src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="https://google.pl"
+      data-qa="big"
+    >
+      <label class="label" for="search">
+        <img
+           src="images/Search.svg"
+           alt="search icon"
+           class="search-icon"
+        >
+      </label>
+      <input
+          id="search"
+          class="input"
+          type="search"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+      >
+    </form>
+
+    <form
+      method="post"
+      action="https://google.pl"
+      data-qa="small"
+    >
+      <label class="label" for="search-small">
+        <img
+           src="images/Search.svg"
+           alt="search"
+           class="search-icon search-icon--small"
+        >
+      </label>
```

Comment:
You should have an empty line between the label and input, same for the above form